libminifi/test/unit/FilePatternTests.cpp

```diff
@@ -8,7 +8,7 @@
  *
  *     http://www.apache.org/licenses/LICENSE-2.0
  *
- * Unless required by applicable law or agreed to in writing, software
+ * Unless CHECKd by applicable law or agreed to in writing, software
```

Comment:
find and replace accident

Reply:
Good catch, updated in c9b464b1282502102a733f473b81fce8821cab94